src/components/TopLine/TopLineChart.tsx

```diff
@@ -0,0 +1,87 @@
+import Chart from "chart.js";
+
+import styles from "./TopLineChart.module.scss";
+
+type Props = {
+    sparklineData: any;
+    pieData: any;
+};
+
+const TopLineChart = (props: Props) => {
+    const { sparklineData, pieData } = props;
+    const divArr = [] as any;
+    for (let i = 0; i < 2; ++i) {
+        const canvasRef = useRef<null | HTMLCanvasElement>(null);
+        useEffect(() => {
+            // Gets a reference to the actual canvas element
+            const canvasElement = canvasRef.current;
+            if (canvasElement) {
+                // Returns an object that provides the methods to draw 2D content to a <canvas>
+                const canvasContext = canvasElement.getContext("2d");
+                if (i == 0) {
+                    new Chart(canvasContext, {
+                        type: "doughnut",
+                        data: pieData,
+                        options: {
+                            legend: {
+                                display: false,
+                            },
+                        },
+                    });
+                } else {
+                    new Chart(canvasContext, {
+                        type: "line",
+                        data: sparklineData,
+                        options: {
+                            legend: {
+                                display: false,
+                            },
+                            scales: {
+                                xAxes: [
+                                    {
+                                        gridLines: {
+                                            drawBorder: false,
+                                            tickMarkLength: 0,
+                                        },
+                                        ticks: {
+                                            display: false,
+                                            maxTicksLimit: 2,
+                                        },
+                                    },
+                                ],
+                                yAxes: [
+                                    {
+                                        gridLines: {
+                                            drawBorder: false,
+                                            tickMarkLength: 0,
+                                        },
+                                        ticks: {
+                                            display: false,
+                                        },
+                                    },
+                                ],
+                            },
+                            elements: {
+                                point: {
+                                    radius: 0,
+                                },
+                            },
+                        },
+                    });
+                }
+                Chart.defaults.scale.gridLines.drawOnChartArea = false;
```

Comment:
These defaults might affect other charts elsewhere on the page, so they should be passed into the `Chart` constructor just above. I think the `drawOnChartArea` should be applied to the `gridLines` object.

I'm not sure what `axisLines` does since I can't find it in the documentation.